src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -284,7 +285,6 @@ class WorkspaceInvitePage extends React.Component {
                             <OptionsSelector
                                 contentContainerStyles={[styles.flexGrow1, styles.flexShrink0, styles.flexBasisAuto]}
                                 listContainerStyles={[styles.flexGrow1, styles.flexShrink1, styles.flexBasis0]}
-                                autoFocus={false}
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/22036. After we re-enabled the auto focus feature the transition lagged and caused a visual glitch. We should have delayed the focus until the transition is complete.